src/persist-client/tests/machine/empty_since

```diff
@@ -133,12 +133,20 @@ error: Since(Antichain { elements: [] })
 # maintenance it needs.
 perform-maintenance
 ----
+v20 ok
```

Comment:
dunno how you did these updates, but wanna take this opportunity to make you aware of the `REWRITE=1` feature of datadriven if you're not already. concretely, if you run something like `REWRITE=1 cargo test -p mz-persist-client --lib`, it will update these in place with the actual answers for you, and then you can eyeball them (saving the copy-paste cycle if you have to update a bunch of the assertions, for example)